src/main/java/core/basesyntax/service/processor/ServiceProcessorOfShopImpl.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax.service.processor;
+
+import core.basesyntax.model.FruitRecord;
+import core.basesyntax.service.strategy.TypeStrategy;
+import java.util.List;
+
+public class ServiceProcessorOfShopImpl implements ServiceProcessorOfShop {
+    private TypeStrategy typeStrategy;
+
+    public ServiceProcessorOfShopImpl(TypeStrategy typeStrategy) {
+        this.typeStrategy = typeStrategy;
+    }
+
+    @Override
+    public void process(List<FruitRecord> transactions) {
+        for (FruitRecord fruitRecord : transactions) {
+            typeStrategy.getType(fruitRecord.getOperation()).apply(fruitRecord);
+        }
+    }
+
+}
```

Comment:
According to the checklist, don't begin class or method implementation with an empty line. Please remove the redundant empty line at the start of the class definition (line 8) and method definition (line 15).